experiment/build/builder.py

```diff
@@ -34,7 +34,7 @@
 from common import logs
 
 from experiment.build import build_utils
-from experiment.run_experiment import DEFAULT_CONCURRENT_BUILDS
+from experiment import run_experiment
```

Comment:
Out of curiosity: Why is importing `run_experiment` preferred over `DEFAULT_CONCURRENT_BUILDS ` here, given we only use the latter?

Reply:
Youre only supposed to import packages/modules, not classes, variables or functions: https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Thanks!